plugins/woocommerce/templates/myaccount/my-address.php

```diff
@@ -60,7 +60,7 @@
 	<div class="u-column<?php echo $col < 0 ? 1 : 2; ?> col-<?php echo $oldcol < 0 ? 1 : 2; ?> woocommerce-Address">
 		<header class="woocommerce-Address-title title">
 			<h3><?php echo esc_html( $address_title ); ?></h3>
-			<a href="<?php echo esc_url( wc_get_endpoint_url( 'edit-address', $name ) ); ?>" class="edit"><?php echo $address ? esc_html__( 'Edit', 'woocommerce' ) : esc_html__( 'Add', 'woocommerce' ); ?></a>
+			<a href="<?php echo esc_url( wc_get_endpoint_url( 'edit-address', $name ) ); ?>" class="edit"><?php echo $address ? esc_html__( 'Edit', 'woocommerce' ) : esc_html__( 'Add', 'woocommerce' ); ?> <?php echo esc_html( $address_title ); ?></a>
```

Comment:
I would recommend using placeholders in the translation string so the words can be reordered if needed e.g. `Edit %s`, using sprintf to inject $address_title.

Reply:
@mikejolley Thanks for the feedback. Please review this commit: [Use placeholder in the edit/add address translation](https://github.com/woocommerce/woocommerce/pull/49171/commits/b2b64ed3c30912fcce7df0796a402bb73397fc11)